backends/python/server/text_embeddings_server/models/default_model.py

```diff
@@ -8,14 +8,16 @@
 
 from text_embeddings_server.models import Model
 from text_embeddings_server.models.types import PaddedBatch, Embedding
+from typing import Optional
 
 tracer = trace.get_tracer(__name__)
 
 
 class DefaultModel(Model):
-    def __init__(self, model_path: Path, device: torch.device, dtype: torch.dtype):
+    def __init__(self, model_path: Path, device: torch.device, dtype: torch.dtype, pooling_mode: Optional[str]):
         model = AutoModel.from_pretrained(model_path).to(dtype).to(device)
         self.hidden_size = model.config.hidden_size
+        self.pooling_mode = pooling_mode
```

Comment:
Can you add mean pooling / CLS pooling forking L46?

Reply:
CLS pooling was already there, I added mean pooling.

What is L46?

Reply:
@OlivierDehaene, I created a [pr](https://github.com/huggingface/text-embeddings-inference/pull/402) to fix this